src/ui/reducers/app.js

```diff
@@ -4,6 +4,8 @@ function initialAppState() {
   return {
     recordingId: null,
     theme: "theme-light",
+    // Whether or not the developer tools toolbox is opened.
```

Comment:
This is open to debate, but I'd prefer not leaving comments on individual state fields unless we think there's something that might be confusing.

For instance, it's possible some of the loading states could be confusing ...

Reply:
Personally, I would prefer to see every state field annotated. I wouldn't easily be able to tell you if `selectedPanel` is a `String` or possibly even a `Number` for the index of the selected panel or the expected object for `tooltip` or the possible values for `theme`.

Boolean states such as splitConsoleOpen could use a bit more structure such as a `is` prefix to make it clear that it's a `Boolean`. Undoubtedly, using typescript for all app state will help as well, but I think leaving the comments like these will at least help somebody in the future whether it is doing the eventual typescript conversion and figuring out the types or someone looking to add a new state field in the future and seeing some existing pattern in the code to follow.